run_release.py

```diff
@@ -1259,6 +1300,20 @@ def _api_key(api_key: str) -> str:
         help="Username to be used when authenticating via ssh",
         type=str,
     )
+    parser.add_argument(
+        "--ssh-key",
+        dest="ssh_key",
+        default=None,
+        help="Path to the SSH key file to use for authentication",
+        type=str,
+    )
+    parser.add_argument(
+        "--security-release",
+        dest="security_release",
+        action="store_true",
+        default=False,
+        help="Indicate this is a security release (only checks for Linux files)",
+    )
```

Comment:
And similarly, we could check the status in the JSON file.

Reply:
Here's a PR to check the status from the devguide, so the RM doesn't have to remember to pass `--security` each time:

https://github.com/pablogsal/release-tools/pull/1

Merging that will update this PR.